UPGRADE-API-1.12.md

```diff
@@ -71,3 +71,31 @@ Here is how the response looks like:
    Wrong parameters otherwise cause empty array `[]` in response and correct parameters return `paymentMethods` available for your `payment`.
 
 1. The 2nd parameter `MetadataInterface` has been removed from `src/Sylius/Bundle/ApiBundle/CommandHandler/Account/ResetPasswordHandler` and replaced by `Sylius\Component\User\Security\PasswordUpdaterInterface` (previously 3rd parameter). From now on a token TTL value must be used instead as the 3rd parameter.
+
+1. The `GET` `api/v2/shop/channels/{code}` endpoint response has been changed and now contains also `defaultLocale`, `locales` and `currencies` fields.
```

Comment:
IMO it is redundant information, in `upgrade file` we should inform about potentially "dangerous" changes like BC breaks or bigger changes in logic.